requirements.txt

```diff
@@ -2,3 +2,4 @@ pandas
 pytest
 setuptools
 pronto
+jug
```

Comment:
This is just needed for building the databases

Reply:
Yes, I'll remove it